prov/opx/configure.m4

```diff
@@ -181,6 +181,7 @@ AC_DEFUN([FI_OPX_CONFIGURE],[
 		])
 	])
 
+	AM_CONDITIONAL([HAVE_CRITERION], [test "x$have_criterion" = "xtrue"])
```

Comment:
Is it ever used in the opx provider?

Reply:
configure fails b/c `HAVE_CRITERION` is not defined in the opx provider. I wasn't able to find a fix for it other than to place that line of code in there.

Reply:
Just in case, the steps to do the rebase (from your topic branch):
```
git pull --rebase upstream main
git push -f
```

Reply:
That's assuming you have configured a remote `upstream`. If not, do the following in advance:
```
git add remote upstream https://github.com/ofiwg/libfabric
```

Reply:
This line should have been removed.

Reply:
@j-xiong all set; thank you for the assistance!